atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -118,8 +118,15 @@ def run(self):
                            IMG_MANIFESTS_PATH, ex)
             self.log.error(msg)
             raise RuntimeError('%s %s' % (msg, ex))
+
+        except Exception as ex:
+            raise RuntimeError('%s ' % (ex))
+
         finally:
-            self.tasker.d.remove_container(container_id)
+            try:
+                self.tasker.d.remove_container(container_id)
+            except Exception as ex:
+                self.log.error(ex)
```

Comment:
This error log could me more descriptive, like `"Failed to remove container %s: %s", container_id, ex`

Reply:
IMHO it should be just warning as it doesn't affect build results

* warning: something happened, shouldn't affect results but may cause issues later
* error: something happened and affect results/tasks
* critical: something happened and system is not able to work properly anymore

Reply:
yop